integration-tests/basic/pom.xml

```diff
@@ -0,0 +1,138 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>io.quarkiverse.cxf</groupId>
+        <artifactId>quarkus-cxf-integration-tests</artifactId>
+        <version>1.5.0-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>quarkus-cxf-integration-test-basic</artifactId>
+
+    <name>Quarkus CXF Extension - Integration Test - Basic</name>
+
+    <dependencies>
+        <dependency>
+            <groupId>io.quarkiverse.cxf</groupId>
+            <artifactId>quarkus-cxf</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-resteasy</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-junit5</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>io.rest-assured</groupId>
+            <artifactId>rest-assured</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.cxf</groupId>
+            <artifactId>cxf-rt-transports-http-netty-server</artifactId>
+            <version>${cxf.version}</version>
+            <scope>test</scope>
+            <exclusions>
+                <exclusion>
+                    <groupId>jakarta.activation</groupId>
+                    <artifactId>jakarta.activation-api</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>jakarta.xml.bind</groupId>
+                    <artifactId>jakarta.xml.bind-api</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>org.apache.geronimo.specs</groupId>
+                    <artifactId>geronimo-jta_1.1_spec</artifactId>
+                </exclusion>
+            </exclusions>
```

Comment:
Hum, aren't these exclusions already defined in the BOM?

Reply:
No, `cxf-rt-transports-http-netty-server` is not in the BOM. It is a test dependency and it does not belong there, IMO. I'd like to remove it altogether in near future and replace it with a testcontainer.

Reply:
Are you sure? I see it here: https://github.com/quarkiverse/quarkus-cxf/pull/488/files#diff-954221b717ff3d93b2f171be5b7d728359dfa4ebee98e8e5691c0697682d247eR155

In fact, that very artifact was the main reason I added the BOM in the first place because I'm using it in my current project (for tests) and it's a maintenance burden to keep it in sync (without it in the BOM).

Reply:
Ah, thanks for the evidence, then I have to correct myself: cxf-rt-transports-http-netty-server is not present in the *flattened* BOM (that gets effectively installed) because it is not a dependency of any of our CXF extensions. I'd prefer to keep that that way, because the BOM should be minimal IMO, covering just the end user runtime use cases.

If you find that particular artifact useful for testing, then I'd vote for putting it into a new quarkus-cxf-bom-test module much like in Quarkus and Camel Quarkus. WDYT?

Reply:
`quarkus-cxf-bom-test` makes sense!

IMO, the first release that contains this change (this PR) has to ship such a test bom as well, otherwise we'd be back to the pre-BOM state as far as that netty artifact is concerned, making it a bit of a breaking change.

cc @shumonsharif